.travis.yml

```diff
@@ -30,6 +30,8 @@ matrix:
     env: CONDA_ENV=py35-dask-dev
   - python: 3.5
     env: CONDA_ENV=py35-pandas-dev
+  - python: 3.5
+    env: CONDA_ENV=py35-mpl2
```

Comment:
let's move this to the `allow_failures` set. This is typically how we handle development builds.